### PR TITLE
Add test case for empty cell #159

### PR DESCRIPTION
**Pull request by vog**

### CODE REVIEW

**Comment by vog** [comment]
This is meant to prepare the fix for #159.

File: tests/test_cell.py [hunk 1]
```diff
@@ -14,6 +14,13 @@ def setUp(self):
         self.book = xlrd.open_workbook(from_this_dir('profiles.xls'), formatting_info=True)
         self.sheet = self.book.sheet_by_name('PROFILEDEF')
         
+    def test_empty_cell(self):
+        sheet = self.book.sheet_by_name('TRAVERSALCHAINAGE')
+        cell = sheet.cell(0, 0)
+        self.assertEqual(cell.ctype, xlrd.book.XL_CELL_EMPTY)
+        self.assertEqual(cell.value, '')
```
Comment (vadmium) [inline tests/test_cell.py]: Maybe I’m missing something, but isn’t comparing to an empty string ('') in Python 2 going to succeed for either kind of string? If you need to ensure it is a Unicode string maybe you should also do self.assertIsInstance(cell.value, unicode), or whatever the time machine equivalent is.

**Comment by vog** [comment]
You are right. The test does not distinguish between unicode strings and byte strings. It was just created so there is *some* test about empty cells.

I'll improve the test in pull request #161.
